### PR TITLE
Added the physical weight of each system to the warning about team lifting.

### PR DESCRIPTION
**Pull request by sambonham**
Also expanded on the warning message based on feedback from Andrew N.
These changes were based on a request made in https://redmine.ixsystems.com/issues/57330

### CODE REVIEW

File: userguide/snippets/es24m_rails.rst [hunk 1]
```diff
@@ -52,10 +52,15 @@ click in place. Repeat this process for the second rear rail.
 Mount the Unit in the Rack
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-**Caution: Two people are required to safely lift the chassis for rack
-installation or removal.** Do not install drives until after the
-chassis has been installed in the rack, and remove all drives before
-removing the chassis from the rack.
+.. caution:: The unit weighs about 40 pounds.
```
Comment (Mrt134) [inline userguide/snippets/es24m_rails.rst]: Spec sheet for ES24 says 40 lbs unloaded, 76 lbs when fully loaded. At minimum, I'd remove "about" -> "The unit weighs 40 pounds." Might want to rework a bit to show the loaded weight too: "The unit weighs 40 pounds empty and 76 pounds when fully loaded with drives."

File: userguide/snippets/es60.rst [hunk 2]
```diff
@@ -139,10 +140,15 @@ rack holes until they lock in place.
 Mount Unit in the Rack
 ~~~~~~~~~~~~~~~~~~~~~~
 
-**Caution: Two people are required to safely lift the chassis for rack
-installation or removal.** Do not install drives until after the
-chassis has been installed in the rack, and remove all drives before
-removing the chassis from the rack.
+.. caution:: The unit weighs about 80 pounds.
```
Comment (Mrt134) [inline userguide/snippets/es60.rst]: As above, spec sheet states 85 lbs unloaded and 175 lbs loaded.

File: userguide/snippets/unified-storage-array.rst [hunk 3]
```diff
@@ -168,9 +168,12 @@ in the rack and secure them with 8 #32 thumbscrews.
 
 The %brand% Storage Array can now be installed into the rack.
 
-.. caution:: Two people are required to safely install or remove the
-   %brand% Storage Array in a rack or cabinet.
-
+.. caution:: The unit weighs about 65 pounds.
```
Comment (Mrt134) [inline userguide/snippets/unified-storage-array.rst]: As above, I don't think "about" is necessary.

File: userguide/snippets/x_rails.rst [hunk 4]
```diff
@@ -23,25 +22,26 @@ Install a spring on the silver posts in the side of each rail.
...
 pins on both ends of the rail with the mounting holes in the rack.
 Swing the clamp latch closed to hold the rail in place. Use two of the
 supplied screws to secure the back end of the rail in place. Repeat
 the process for the second rail.
 
-
 .. figure:: images/tn_x_railclamp.png
    :width: 100%
 
+.. caution:: The unit weighs about 30 pounds.
```
Comment (Mrt134) [inline userguide/snippets/x_rails.rst]: Same as above, exact weight from the spec sheet says 26 lbs unloaded, 44 lbs loaded.

File: userguide/snippets/es24m_rails.rst (hunk 1 above)
Comment (Mrt134) [reply userguide/snippets/es24m_rails.rst]: Reviewed M-Series BSG and noticed it uses this file for the rack mount instructions too. This caution needs to show weights for both the ES24 and the M-Series.

Comment (wblock) [reply userguide/snippets/es24m_rails.rst]: There is some margin for error and changes due to packaging.  The "about" is good to show that some variation is expected and the quoted weights should be verified before someone ships one.

**Comment by wblock** [comment]
Nobody knows where the idea to put exact weights came from.  Due to potentially misleading the reader, and because we already have warnings that two people are required to rack a system, I'm closing this PR.  If it is resubmitted, we can follow up with the requester.

**Pull request closed without merging**